prow/plugins/updateconfig/updateconfig_test.go

```diff
@@ -1200,6 +1200,66 @@ func TestUpdateConfig(t *testing.T) {
 	}
 }
 
+func TestFilterChanges(t *testing.T) {
```

Comment:
If you want to write `TestFilterChanges` I would expect to see tests for all valid use-cases of `FilterChanges`. I would also like to see the more targeted tests for `handleDefaultNamespace`

Reply:
Done.